tests/unit/test_results.py

```diff
@@ -0,0 +1,68 @@
+"""Module to test the KytosGraph in graph.py."""
+from unittest import TestCase
+
+from kytos.core.interface import Interface
+from kytos.core.link import Link
+from kytos.core.switch import Switch
+
+# module under test
+from graph import KytosGraph
+
+
+class TestResults(TestCase):
+    """Tests for the graph class."""
+
+    def setUp(self):
+        """Setup for most tests"""
+        switches, links = self.generate_topology()
+        self.graph = KytosGraph()
+        self.graph.clear()
+        self.graph.update_nodes(switches)
+        self.graph.update_links(links)
+
+    def get_path(self, source, destination):
+        """Return the shortest path"""
+        results = self.graph.shortest_paths(source, destination)
+        return results
+
+    def get_path_constrained(self, source, destination, maximum_misses=None,
+                             **metrics):
+        """Return the constrained shortest path"""
+        return self.graph.constrained_flexible_paths(source, destination,
+                                                     maximum_misses,
+                                                     **metrics)
+
+    @ staticmethod
```

Comment:
@MarvinTorres, can you change these decorators? Just to standardize.

```suggestion
    @staticmethod
```

Reply:
Okay, will do. Thank you!